src/mapgen_v6.cpp

```diff
@@ -58,14 +58,14 @@ FlagDesc flagdesc_mapgen_v6[] = {
 MapgenV6::MapgenV6(int mapgenid, MapgenV6Params *params, EmergeManager *emerge)
 	: Mapgen(mapgenid, params, emerge)
 {
-	this->m_emerge = emerge;
-	this->ystride = csize.X; //////fix this
+	m_emerge = emerge;
```

Comment:
I guess you mean use 'emerge' directly because it is present here `: Mapgen(mapgenid, params, emerge)`?
This is the same as in mapgenbasic which was coded recently https://github.com/minetest/minetest/blob/master/src/mapgen.cpp#L543